packages/js/src/components/Modal.tsx

```diff
@@ -76,9 +99,12 @@ export default function Modal({
         <div
           ref={modalRef}
           className={cn(
+            "fb-bottom-0 fb-inset-x-0",
             getPlacementStyle(placement),
             show ? "fb-opacity-100" : "fb-opacity-0",
-            "fb-h-fit fb-pointer-events-auto fb-absolute fb-w-full fb-max-w-sm fb-overflow-hidden fb-rounded-lg fb-bg-white fb-shadow-lg fb-ring-1 fb-ring-black fb-ring-opacity-5 fb-transition-all fb-duration-500 fb-ease-in-out sm:fb-m-4"
+            "fb-h-fit fb-pointer-events-auto fb-absolute fb-w-full sm:fb-max-w-sm fb-overflow-hidden fb-rounded-lg fb-bg-white fb-shadow-lg fb-ring-1 fb-ring-black fb-ring-opacity-5 fb-transition-all fb-duration-500 fb-ease-in-out sm:fb-m-4",
```

Comment:
Here fb-bottom-0 and fb-inset-x-0 makes the modal sticks to bottom on small devices